tox.ini

```diff
@@ -9,6 +9,7 @@ deps =
     django111: Django>=1.11,<1.12
     django20: Django>=2.0,<2.1
     django21: Django>=2.1,<2.2
+    py27: mock==2.0.0
```

Comment:
As a bonus point, can you remove `test_requires` from `setup.py` ? It's broken in various ways : it has a typo - it should be `tests_require` - and since we need extra settings it doesn't work out of the box, not to mention the fact that it needs a different version of django depending on the version of python being used.